src/compiler/binder.ts

```diff
@@ -210,6 +216,24 @@ namespace ts {
             return propAccess.name.text;
         }
 
+        /**
+         * Returns the first invocation of an identifier 'require' with one argument,
+         * or undefined if that none exist.
+        */
+        function findCommonJSRequireCalls(node: Node): Node {
+            if (node.kind === SyntaxKind.CallExpression) {
+                let call = <CallExpression>node;
+                if(call.expression.kind === SyntaxKind.Identifier &&
+                    (<Identifier>call.expression).text === 'require' &&
+                    call.arguments.length === 1) {

```

Comment:
Actually, never mind, since this is just for ensuring module scoping rules, right?